g_appcode.go

```diff
@@ -1062,7 +1062,15 @@ func GetAll{{modelName}}(query map[string]string, fields []string, sortby []stri
 	for k, v := range query {
 		// rewrite dot-notation to Object__Attribute
 		k = strings.Replace(k, ".", "__", -1)
-		qs = qs.Filter(k, v)
+		if strings.Contains(k, "isnull") {
+			var v2 bool
+			if v == "true" || v == "1" {
+				v2 = true
+			}
+			qs = qs.Filter(k, v2)
```

Comment:
@r0nni3 The inner if statement can be replaced with: `qs = qs.Filter(k, (v == "true" || v == "1"))`